in_progress/mission-statement.md

```diff
@@ -0,0 +1,58 @@
+# Mission Statement
+
+## Goals
+
+We want to make single cell omics data analysis as accessible as possible. This means
```

Comment:
Note that our slogan is broader. Should this be more general, but also state here that we start with single-cell stuff?